src/openrct2/interface/Window.cpp

```diff
@@ -1239,26 +1241,26 @@ void window_draw_viewport(rct_drawpixelinfo* dpi, rct_window* w)
     viewport_render(dpi, w->viewport, dpi->x, dpi->y, dpi->x + dpi->width, dpi->y + dpi->height);
 }
 
-void window_set_position(rct_window* w, int32_t x, int32_t y)
+void window_set_position(rct_window* w, ScreenCoordsXY screenCoords)
 {
-    window_move_position(w, x - w->x, y - w->y);
+    window_move_position(w, ScreenCoordsXY(screenCoords.x - w->x, screenCoords.y - w->y));
 }
 
-void window_move_position(rct_window* w, int32_t dx, int32_t dy)
+void window_move_position(rct_window* w, ScreenCoordsXY screenCoords)
```

Comment:
should make this clear that screenCoords is a delta value. Suggest renaming this `deltaCoords`.

Reply:
Done!